qemu/tests/single_driver_install.py

```diff
@@ -0,0 +1,156 @@
+import re
+import logging
+from virttest import utils_misc
+from virttest import error_context
+from avocado.core import exceptions
+from aexpect import ShellCmdError
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    This Test is mainly used as subtests
+    1) Boot up VM
+    2) Uninstall driver (Optional)
+    3) Reboot vm (Based on step 2)
+    4) Update / Downgrade / Install driver
+    5) Reboot vm
+    6) Backup driver installation logs
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    def show_file_extentions(session_serial):
+        """
+        Show file extentions in windows guest.
+
+        :param session_serial: VM session.
+        """
+        show_file_ext_cmd = params.get("show_file_ext_cmd")
+        session_serial.cmd(show_file_ext_cmd)
+
+    def get_installation_logs(session_serial):
+        """
+        Get driver installation log.
+
+        :param session_serial: VM session.
+        """
+        tmp_debug = "%systemroot%\\DPINST.log,\
+                     %systemroot%\\inf\\setupapi.dev.log"
+        driver_logs = params.get("driver_debug_file", tmp_debug).split(',')
+        for _file in driver_logs:
+            status = session_serial.cmd_status("dir %s" % _file)
+            if not status:
+                output = session_serial.cmd("type %s" % _file)
+                _file_host = _file.split("\\")[-1].strip()
```

Comment:
use raw string to save file path, then you can get base-name of the file like:
import re
re.split(r"\*", '%r' % file_path)[-1]